lib/db/mysql/schema.sql

```diff
@@ -43,3 +43,14 @@ CREATE TABLE IF NOT EXISTS tokens (
   scope VARCHAR(256) NOT NULL,
   createdAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 ) ENGINE=InnoDB CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+
+CREATE TABLE IF NOT EXISTS developers (
+  email VARCHAR(256) NOT NULL PRIMARY KEY,
```

Comment:
Separately, we wanted to keep developers by email, instead of `userId` from fxa?